loleaflet/src/layer/tile/TileLayer.js

```diff
@@ -2168,7 +2168,25 @@ L.TileLayer = L.GridLayer.extend({
 	},
 
 	_tileOnLoad: function (done, tile) {
-		done(null, tile);
+		// slurp multiple async tile loads into one render:
+		// FIXME: probably we should nicely combine
+		// async image decoding into Socket.js' _slurpMessage
+		// to save another 1ms timer.
+		var that = this;
+		if (!that._slurpQueue || !that._slurpQueue.length) {
+			setTimeout(function() {
+				console.log2('Slurp tiles ' + that._slurpQueue.length / 2);
```

Comment:
we should comment that out on all branches, after verifying that it is getting similar numbers to the above =) otherwise great. Thanks Gokay do merge on your own.